content/general/authentication.textile

```diff
@@ -226,176 +235,59 @@ auth.requestToken({ capability: {
 
-The resulting string must then the UTF8-encoded and then HMAC value is computed with "hmac-sha-256":http://tools.ietf.org/html/rfc4868 using the key secret value.
+We encourage customers to always issue tokens to clients so that they authenticate using the short-lived token and do not have access to a customer's private API keys. Since the customer can then control the @client ID@ that may be used by any of its clients, all other clients can rely on the validity of the @client ID@ in published messages and of members present in presence channels.
 
-The HMAC value is then "base-64 encoded":https://tools.ietf.org/html/rfc4648.
+The following Javascript example demonstrates how to issue a token with an explicit @client ID@ that, when used by a client, will then be considered an *identified client*.
 
-h3. Request body format
-
-In the case of a signed token request, the request body is the JSON stringified representation of the object with the form:
-
-bc[json]. {
-  keyName: "<app ID>:<key ID>",
-  ttl: <expiry in milliseconds>,
-  capability: "<capability string>",
-  clientId: "<client ID optional>",
-  timestamp: <timestamp as ms since epoch>,
-  nonce: "<random unique nonce>",
-  mac: "<base 64-encoded HMAC value>"
-}
-
-An unsigned token request is identical except that the mac property is omitted. Note that "Basic authentication":#basic-authentication must be used in order to request a token with an unsigned request.
-
-h3. Response body format
-
-If successful, the authorization request returns the JSON stringified representation of an object containing the token:
-
-bc[json]. {
-  token: "<token value>",
-  issued: <timestamp as ms since epoch>,
-  expires: <timestamp as ms since epoch>,
-  capability: "<canonical capability text>",
-  clientId: "<client ID optional>"
-}
-
-h3. Example token requests
-
-h4. Unsigned token request example
-
-bc[sh]. curl -X POST "https://rest.ably.io/keys/{{API_KEY_NAME}}/requestToken" \
- --user "{{API_KEY}}" \
- --header "Content-Type: application/json" \
- --data '{
-   "keyName": "{{API_KEY_NAME}}",
-   "ttl": "3600000",
-   "capability":
-     "{\"private\":[\"subscribe\",\"publish\",\"presence\"],\"*\":[\"subscribe\"]}",
-   "clientId": "unique_identifier",
-   "timestamp": {{MS_SINCE_EPOCH}},
-   "nonce": "95e543b88299f6bae83df9b12fbd1ecd"
-}'
-
-Responds with JSON token:
-
-bc[json]. {
-  "token": "{{APP_ID}}.HHZNjgqmC-ACW....truncated",
-  "keyName": "{{API_KEY_NAME}}",
-  "issued": 1449745478956,
-  "expires": 1449749078956,
-  "capability":
-    "{\"*\":[\"subscribe\"],\"private\":[\"presence\",\"publish\",\"subscribe\"]}",
-  "clientId": "unique_identifier"
-}
+```[javascript](code-editor:realtime/auth-client-id)
+  var realtime = new Ably.Realtime({ key: '{{API_KEY}}' });
```

Comment:
Might consider using the Rest client for consistency with the other serverside examples